hack/make.sh

```diff
@@ -81,6 +81,14 @@ if command -v git &> /dev/null && git rev-parse &> /dev/null; then
 	GITCOMMIT=$(git rev-parse --short HEAD)
 	if [ -n "$(git status --porcelain --untracked-files=no)" ]; then
```

Comment:
just a nit: but perhaps we should've kept this result in a var, so that we can simply "echo" that below?